activemodel/lib/active_model/attribute_methods.rb

```diff
@@ -417,6 +430,42 @@ def define_proxy_call(code_generator, name, proxy_target, parameters, *call_args
           end
         end
 
+        # Generates body that calls the original method directly.
```

Comment:
Elsewhere I've just recommended that we avoid some of these constructs entirely by calling existing methods... for those that remain (or even if we can't do that, and have to keep all of the below behaviour), I think they're short enough (and no longer overridden anywhere, which I think might've been the original motivation for extraction?) that it's better for readability to keep this all in-line in the calling method.